README.rst

```diff
@@ -104,3 +130,45 @@ Apply the changes:
 You should have requested number of resources spawned on Openstack, and ansible_inventory file produced as output for Kayobe.
 
 Copy your generated id_rsa and id_rsa.pub to ~/.ssh/ on Ansible control host if you want Kayobe to automatically pick them up during bootstrap.
+
+Configure Ansible Control Host
+
+Using the `deploy-oc-networks.yml` playbook you can setup the ansible control host to include the kayobe/kayobe-config repositories with `hosts` and `admin-oc-networks`.
+It shall also setup the kayobe virtual environment, allowing for immediate configure and deployment of OpenStack.
+
+First you must ensure that you have `Ansible installed <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_ on your local machine.
+
+.. code-block:: console
+
+   pip install --user ansible
+
+Secondly if the machines are behind an SSH bastion you must ensure that your ssh config is setup appropriately with a proxy jump
+
+.. code-block:: console
+
+   Host lab-bastion
+      HostName BastionIPAddr
+      User username
+      IdentityFile ~/.ssh/key
+
+   Host 10.*
+      ProxyJump=lab-bastion
+      ForwardAgent no
+      IdentityFile ~/.ssh/key
+      UserKnownHostsFile /dev/null
+      StrictHostKeyChecking no
+
+Install the ansible requirements.
+
+.. code-block:: console
+
+   ansible-galaxy install -r ansible/requirements.yml
+
+Finally run the ansible playbooks. 
+You may need to run `grow-control-host.yml` if you are using LVM images and the LVMs are too small to install Ansible.
+If not you can skip that playbook and proceed onto `deploy-openstack-config` which shall configure your Ansible control host in preparation for deployment.
+
+.. code-block:: console
+
+   ansible-playbook -i ${ansible_ip}, ansible/grow-control-host.yml -e ansible_user=centos
+   ansible-playbook -i ${ansible_ip}, ansible/deploy-openstack-config.yml -e ansible_user=centos
```

Comment:
No mention of deploy-openstack.sh or tear-down.sh

Reply:
These sections have been provided below this comment